package.json

```diff
@@ -75,6 +75,7 @@
         }
       ],
       "quotes": "off",
+      "jsx-quotes": ["warn", "prefer-double"],
```

Comment:
prefer-single

Reply:
Single is 
<img width="286" alt="image" src="https://user-images.githubusercontent.com/25384290/184088098-9234aa35-3530-47be-8efb-1f2211623751.png">